ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -310,9 +351,9 @@ export const AdminInstitutionEdit = withUrlParams()(class extends React.Componen
                 <BulletAlignedUnorderedList>
                   {errors.displayName && <li>Display Name should be of at most 80 Characters</li>}
                   {errors.organizationTypeEnum && <li>Organization Type should not be empty</li>}
-                  {errors.duaTypeEnum && <li>Agreement Type should not be empty</li>}
-                  {!errors.duaTypeEnum && errors.emailDomain && <li>Email Domain should not be empty</li>}
-                  {!errors.duaTypeEnum && errors.emailAddresses && <li>Email Address should not be empty</li>}
+                  {errors.tierRequirements && <li>Agreement Type should not be empty</li>}
+                  {!errors.tierRequirements && errors.tierEmailDomains && <li>Email Domain should not be empty</li>}
```

Comment:
should probably make these plural

Reply:
make domains and addresses as plural, not changing tierRequirement for now.